gulpfile.js

```diff
@@ -127,6 +128,13 @@ gulp.task('styles', function stylesTask() {
     .pipe(gulp.dest(DEST_PATH + 'styles'));
 });
 
+gulp.task('sass-lint', function sassLintTask() {
+  return gulp.src(SRC_PATH+ '/styles/**/*.scss')
```

Comment:
Me thinks you still need a space before this `+` operator, otherwise I was getting this error locally:
```
Message:
    Infix operators must be spaced.
Details:
    fileName: gulpfile.js
    lineNumber: 132
```